spec/jekyll-contentful/data_exporter_spec.rb

```diff
@@ -75,7 +75,7 @@
 
       it 'serializes entries onto data file' do
         allow(File).to receive(:open).and_yield(StringIO.new)
-        expect_any_instance_of(::Jekyll::Contentful::Serializer).to receive(:to_yaml)
+        expect(YAML).to receive(:dump)
```

Comment:
Hey @felixjung,

It was ok in your context, so no worries about that, the thing is that the Contentful Serializer doesn't do the same as a simple `YAML.dump`, as it runs through either the default Mapper, or through user defined Mappers, which transform the resulting YAML however the Mapper is set.

Therefore, removing the Serializer is something we don't want